internal/connector/pingone_platform/pingone_platform_connector.go

```diff
@@ -76,6 +76,7 @@ func (c *PingonePlatformConnector) Export(format, outputDir string, overwriteExp
 		resources.BrandingSettingsResource(&c.clientInfo),
 		resources.BrandingThemeResource(&c.clientInfo),
 		resources.BrandingThemeDefaultResource(&c.clientInfo),
+		resources.CertificateResource(&c.clientInfo),
```

Comment:
Not a problem with this resource, but I'm realizing with Go's recommended naming, it might make more sense to leave the "Resource" suffix off of these. See https://go.dev/doc/effective_go#package-names

Reply:
Sounds good. I'll take a look at that and include it in a feedback PR.